### PR TITLE
Code review feature pull request pull request #1

### PR DESCRIPTION
**Pull request by efmomoh**
This is a code review project. In this project, a youtube page-like was built using

- [ ] HTML
- [ ] CSS
- [ ] Readme file, and
- [ ] Linters file for clean codes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </nav>
+    </section>
+    <!-- nav-link-end  -->
+    <section>
+        <div  class="flex-container">
+            <ul class="left-bar-video" >
+                <li>
+                    <iframe width="180" height="110" src="https://www.youtube.com/embed/l5yNJWKnoDg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </li>
+            </ul>
```
Comment (efmomoh) [inline index.html:61]: - [OPTIONAL] Kindly add a short description to the playlist that will better help our users to understand what content of the video it is. 🤓

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                    <iframe width="150" height="100" src="https://www.youtube.com/embed/XqhRCxjRmrM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </li>
+                <li>
+                    <iframe width="150" height="100" src="https://www.youtube.com/embed/XqhRCxjRmrM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </li>
+                <li>
+                    <iframe width="150" height="100" src="https://www.youtube.com/embed/XqhRCxjRmrM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </li>
+            </ul>
+        </ul>
```
Comment (efmomoh) [inline index.html:88]: - [OPTIONAL] Please add pictures to the list as well so as to make our project look good 👍

**Review by efmomoh — COMMENTED** [review]
CONGRATULATIONS 🎉🎉🎉🎉🏆🏆🏆🏆🏆🏆🏆🏆
So far you have done a good job 👍👏👏👏👏👏👏
What else is remaining? Absolutely, nothing! It's time to merge our branch.

Successfully passed the following checks:

- [ ] HTML ✔️
- [ ] CSS ✔️
- [ ] linters ✔️
- [ ] README.md ✔️

**Review by Cabraham1 — APPROVED** [review]
Hi @efmomoh  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good use of inline comment ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**